README.md

```diff
@@ -43,103 +43,118 @@ Dedicated servers hosted on Steam are usually required to be running the *latest
-| Left 4 Dead 2 | [`sourceservers/left4dead2`][srcds-left4dead2-dockerhub-link] | [![srcds-left4dead2-version-badge][]][srcds-left4dead2-metadata-link] | [![srcds-left4dead2-layers-badge][]][srcds-left4dead2-metadata-link] |
-| Team Fortress 2 | [`sourceservers/tf`][srcds-tf-dockerhub-link] | [![srcds-tf-version-badge][]][srcds-tf-metadata-link] | [![srcds-tf-layers-badge][]][srcds-tf-metadata-link] |
+| Game | Image | Tag | Size | Layers |
+|:-:|:-:|:-:|:-:|:-:|
+| Counter-Strike: Global Offensive | [`sourceservers/csgo`][srcds-csgo-dockerhub-link] | [![srcds-csgo-version-badge][]][srcds-csgo-metadata-link] | [![srcds-csgo-size-badge][]][srcds-csgo-metadata-link] | [![srcds-csgo-layers-badge][]][srcds-csgo-metadata-link] |
+| Counter-Strike: Source | [`sourceservers/cstrike`][srcds-cstrike-dockerhub-link] | [![srcds-cstrike-version-badge][]][srcds-cstrike-metadata-link] | [![srcds-cstrike-size-badge][]][srcds-cstrike-metadata-link] | [![srcds-cstrike-layers-badge][]][srcds-cstrike-metadata-link] |
+| Day of Defeat: Source | [`sourceservers/dod`][srcds-dod-dockerhub-link] | [![srcds-dod-version-badge][]][srcds-dod-metadata-link] | [![srcds-dod-size-badge][]][srcds-dod-metadata-link] | [![srcds-dod-layers-badge][]][srcds-dod-metadata-link] |
+| Half-Life 2: Deathmatch | [`sourceservers/hl2mp`][srcds-hl2mp-dockerhub-link] | [![srcds-hl2mp-version-badge][]][srcds-hl2mp-metadata-link] | [![srcds-hl2mp-size-badge][]][srcds-hl2mp-metadata-link] | [![srcds-hl2mp-layers-badge][]][srcds-hl2mp-metadata-link] |
+| Left 4 Dead | [`sourceservers/left4dead`][srcds-left4dead-dockerhub-link] | [![srcds-left4dead-version-badge][]][srcds-left4dead-metadata-link] | [![srcds-left4dead-size-badge][]][srcds-left4dead-metadata-link] | [![srcds-left4dead-layers-badge][]][srcds-left4dead-metadata-link] |
+| Left 4 Dead 2 | [`sourceservers/left4dead2`][srcds-left4dead2-dockerhub-link] | [![srcds-left4dead2-version-badge][]][srcds-left4dead2-metadata-link] | [![srcds-left4dead2-size-badge][]][srcds-left4dead2-metadata-link] | [![srcds-left4dead2-layers-badge][]][srcds-left4dead2-metadata-link] |
+| Team Fortress 2 | [`sourceservers/tf`][srcds-tf-dockerhub-link] | [![srcds-tf-version-badge][]][srcds-tf-metadata-link] | [![srcds-tf-size-badge][]][srcds-tf-metadata-link] | [![srcds-tf-layers-badge][]][srcds-tf-metadata-link] |
 
 [srcds-csgo-dockerhub-link]: https://hub.docker.com/r/sourceservers/csgo
 [srcds-csgo-version-badge]: https://images.microbadger.com/badges/version/sourceservers/csgo.svg
-[srcds-csgo-layers-badge]: https://images.microbadger.com/badges/image/sourceservers/csgo.svg
-[srcds-csgo-metadata-link]: https://microbadger.com/images/sourceservers/csgo
+[srcds-csgo-size-badge]: https://img.shields.io/docker/image-size/sourceservers/csgo/latest?label=&style=flat-square
```

Comment:
@joeltimothyoh to me, the PR is perfectly fine. The table column is named "Tag", and the "latest" badges look perfectly fine. I don't think `img.shields.io` will change the `/docker/v/` badge, so we can already just merge this PR and stop using `microbadger`.

Reply:
You don't understand. The game images table has all along only used `microbadger.com` badges. This PR does not include using the `img.shields.io/docker/v/` badge. The tags appear fine because because badges you're seeing still use the `images.microbadger.com/badges/version` badge. Pay attention @leojonathanoh

Reply:
@joeltimothyoh my bad. I thought the third column was `img.shields.io/docker/image-size`. As I said the `img.shields.io/docker/v/` is misleading. So it might be better to just do away with the column.

Reply:
Which is why I’m thinking of just keeping the version badges like how they’ve been since the tags are a nice to have in the table. Their links point to `hub.docker` though which I think marks the transition to referring to image repositories instead as sources of truth.

PR as it is looks alright as you say. We could open another one for revising the versions badges just before the `microbadger.com` shutdown. What you do think @leojonathanoh

Reply:
> PR as it is looks alright as you say. We could open another one for revising the versions badges just before the microbadger.com shutdown. What you do think @leojonathanoh

@joeltimothyoh ok

Reply:
lets just merge this PR